### DISCUSSION TITLE
error when I build kade engine

### DISCUSSION THREAD

**Discussion by lostinstruments** [KadeArchive/Kade-Engine#378] [Q&A] [answered]
Everytime I try to build the kade engine I get this error:

"C:/HaxeToolkit/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 10-23 : Warning : This typedef is deprecated in favor of haxe.xml.Access
C:/HaxeToolkit/haxe/lib/polymod/git/polymod/format/XMLMerge.hx:189: characters 30-43 : Warning : This typedef is deprecated in favor of haxe.xml.Access
source_clown/PlayState.hx:291: characters 3-39 : Warning : `FlxCamera.defaultCameras` is deprecated, use `FlxG.cameras.setDefaultDrawTarget` instead
Sys_error("exportCLOWN/release/windows/obj/Options.txt: Permission denied")"

I'm building using windows and haxe version 4.1.5. I have also built it to html5 and it works fine, it's just whenever I build it to windows I get this error.

**Comment by Torre939** [comment]
An Error ahhhhh but my phone only glitches i never discovered an error on my phone

**Comment by Torre939** [comment]
An Error ahhhhh but my phone only glitches i never discovered an error on my phone

**Comment by isakube** [comment]
Those first two about XMLMerge.hx are normal and are just warnings. The third is also fine I think. I don't have any idea about the fourth though.

**Comment by puyoxyz** [comment, marked as answer]
it looks like you don't have permission to access the file(s)? make sure you have kade engine *inside of your user directory*, like in your Documents folder or your Desktop

**Reply by lostinstruments** [reply]
At first kade engine was on my desktop, but even when I moved it to the documents folder it still gives me the same error. I reverted all the code to before I changed anything and I still get the same error.

**Reply by puyoxyz** [reply]
send output of `haxelib list` and `haxe --version` pls

**Reply by lostinstruments** [reply]
It's working now actually! I switched to kade engine 1.4 but what I think what was happening had to do with my antivirus software I have? it kept stopping it from building for some reason (i really should just delete it gets so annoying lol) so I said to allow it and its building right now. Thank you everyone regardless I hope everyone has a nice day!

**Comment by Torre939** [comment]
How to fix errors in windows
